doc/getting-started/building-the-node-on-windows.md

```diff
@@ -0,0 +1,110 @@
+# Development build of Cardano Node on Windows
+
+This document describes how to build a __DEVELOPMENT__ version of `cardano-node`.
+This is not a __PRODUCTION__ build!
+
+Building on `Windows` is a non trivial task.  There are various tools one needs
+to install and configure.  We recommend to use
+[chocolatey](https://chocolatey.org) (which provides the `choco` command) to
+install them.
+
+All instructrions that invoke `choco` can be done in a PowerShell with root
+previlages.  For development work we recommend installing and using `git-bash`
+(with `choco install git-bash`).
```

Comment:
I am getting this error when executing `choco install git-bash`

`git-bash not installed. The package was not found with the source(s) listed.
 Source(s): 'https://chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://chocolatey.org/docs/troubleshooting for more
 assistance.`

Reply:
choco is installed correclty 
`PS C:\WINDOWS\system32> choco
Chocolatey v0.10.15`

Reply:
It's just `choco install git`